zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -119,6 +128,14 @@ public static boolean executePublish(Collection collection, CollectionReader col
             success &= publishDatasets(collection);
         }
 
+        if (imageFuture != null) {
```

Comment:
This is how I had it initially, but then the linting was moaning because I was not checking for a null before doing `imageFuture.get()`. If I put it under a feature flag instead, should I just ignore the null (im)possibility or have an exception to throw that will never occur?

Reply:
I have updated the code to use the feature flag and throw an exception on nulls instead.